paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentElement.kt

```diff
@@ -20,17 +19,19 @@ import com.stripe.android.model.PaymentMethod
 import com.stripe.android.paymentsheet.PaymentMethodsUI
 import com.stripe.android.paymentsheet.R
 import com.stripe.android.paymentsheet.forms.FormFieldValues
+import com.stripe.android.paymentsheet.injection.FormViewModelSubcomponent
 import com.stripe.android.paymentsheet.paymentdatacollection.FormArguments
 import com.stripe.android.paymentsheet.paymentdatacollection.ach.USBankAccountForm
 import com.stripe.android.paymentsheet.paymentdatacollection.ach.USBankAccountFormArguments
-import com.stripe.android.paymentsheet.viewmodels.BaseSheetViewModel
 import com.stripe.android.ui.core.forms.resources.LpmRepository
 import com.stripe.android.uicore.image.StripeImageLoader
 import kotlinx.coroutines.flow.Flow
+import javax.inject.Provider
 
 @Composable
 internal fun PaymentElement(
-    sheetViewModel: BaseSheetViewModel,
+    formViewModelSubComponentBuilderProvider: Provider<FormViewModelSubcomponent.Builder>,
+    processing: Boolean,
     enabled: Boolean,
```

Comment:
Should we remove enabled? It's always the opposite of processing.

Reply:
I did the other way around, kept enabled, and removed processing.